
    <!doctype html>

    <html lang="en">
    <head>
        <meta charset="utf-8">    
        <script type="text/JavaScript" src="https://MomentJS.com/downloads/moment.js"></script>
        <script type="text/JavaScript" src="https://momentjs.com/downloads/moment-timezone-with-data.min.js"></script>
        <title>Cerebellar nuclei evolved by repeatedly duplicating a conserved cell-type set</title>
        <style>
            
    * {
        font-family: "Trebuchet MS", Helvetica, sans-serif;
    }
    
        </style>
        <script type="text/JavaScript">
            
	function LT(t) {
        var m = moment.utc(t).tz(moment.tz.guess());
		document.write(m.format('MMMM Do YYYY, HH:mm z'));
	}
    
        </script>
    </head>
    <body>
        <h3>
            <a href="https://neuromatch.io">Neuromatch</a> 3 /
            <script type="text/JavaScript">LT("2020-10-27 16:00");</script>
            /
            Track 2
            /
            Traditional talk
        </h3>
        <h1>Cerebellar nuclei evolved by repeatedly duplicating a conserved cell-type set</h1><h2>Justus Kebschull</h2><h3>Justus M Kebschull, Stanford University; Noam Ringach, Stanford University; Ethan B Richman, Stanford University; Drew Friedmann, Stanford University; Sai Saroja Kolluru, Chan Zuckerberg Biohub; Robert C Jones, Stanford University; William E Allen, Harvard Univeristy; Ying Wang, UC Davis; Huaijun Zhou, UC Davis; Seung Woo Cho, Stanford University; Howard Y Chang, HHMI and Stanford University; Karl Deisseroth, HHMI and Stanford University; Stephen R Quake, Chan Zuckerberg Biohub and Stanford University; Liqun Luo, HHMI and Stanford University;</h3><h2>Abstract</h1><p>How have complex brain regions, circuits, and cell types evolved from simple origins? Here we investigate brain region evolution at cell-type resolution in the cerebellar nuclei, the output structures of the cerebellum. We applied single-nucleus RNA sequencing in chickens, mice, and humans, STARmap spatial transcriptomic analysis in chicken and mice, and whole-CNS projection mapping in mice. Our work revealed a conserved cell type set containing three classes of region-invariant inhibitory neurons and two classes of region-specific excitatory neurons. This cell type set forms an archetypal cerebellar nucleus that was repeatedly duplicated to create new regions. In excitatory neurons, duplication was accompanied by divergence in gene expression and shifts in projection patterns (neofunctionalization). By contrast, inhibitory neurons maintained their gene expression signatures (isofunctionalization). Interestingly, the excitatory cell class that preferentially funnels information to lateral frontal cortices in mice becomes predominant in the massively expanded human Lateral CN. Our data provide the first characterization of CN transcriptomic cell types in three species and suggest a model of brain region evolution by duplication and divergence of entire cell type sets.</p>
    </body>
    </html>
    