
    <!doctype html>

    <html lang="en">
    <head>
        <meta charset="utf-8">    
        <script type="text/JavaScript" src="https://MomentJS.com/downloads/moment.js"></script>
        <script type="text/JavaScript" src="https://momentjs.com/downloads/moment-timezone-with-data.min.js"></script>
        <title>In vivo Perturb-seq: study gene function at scale</title>
        <style>
            
    * {
        font-family: "Trebuchet MS", Helvetica, sans-serif;
    }
    
        </style>
        <script type="text/JavaScript">
            
	function LT(t) {
        var m = moment.utc(t).tz(moment.tz.guess());
		document.write(m.format('MMMM Do YYYY, HH:mm z'));
	};
    function time_between(start, end) {
        var s = moment.utc(start);
        var e = moment.utc(end);
        var now = moment();
        return (s<=now) && (now<=e);
    };
    function update_visibility() {
        var now = moment();
        var elems = document.getElementsByClassName("visible_at_time");
        for(var i=0; i<elems.length; i++) {
            s = moment.utc(elems[i].dataset.start);
            e = moment.utc(elems[i].dataset.end);
            if ( (s<=now) && (now<=e) ) {
                elems[i].style.display = "block";
            } else {
                elems[i].style.display = "none";
            }
        }
    };
    setInterval(update_visibility, 60*1000);
    
        </script>
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    </head>
    <body>
        <h3>
            <a href="https://neuromatch.io">Neuromatch</a> 3 /
            <script type="text/JavaScript">LT("2020-10-27 13:30");</script>
            /
            Track 1
            /
            Traditional talk
            <div class="visible_at_time" data-start="2020-10-27 13:30" data-end="2020-10-27 13:45">
                <a href='https://www.youtube.com/watch?v=OBmtAVSWc08'><i class="fa fa-youtube-play" style="font-size:24px;color:red"></i></a>
                <a href='https://www.youtube.com/watch?v=OBmtAVSWc08'>Watch now on YouTube</a>
            </div>
        </h3>
        <h1>In vivo Perturb-seq: study gene function at scale</h1><h2>Xin Jin</h2><h3>Xin Jin, Society of Fellows, Harvard University; Sean K. Simmons, Broad Institute of MIT and Harvard; Ashwin S. Shetty, Department of Stem Cell and Regenerative Biology, Harvard University; Joshua Z. Levin, Broad Institute of MIT and Harvard; Aviv Regev, Broad Institute of MIT and Harvard, MIT; Feng Zhang, Broad Institute of MIT and Harvard, McGovern Institute of Brain Science, MIT; Paola Arlotta, Department of Stem Cell and Regenerative Biology, Harvard University</h3><h2>Abstract</h1><p>The number of disease risk genes and loci identified through human genetic studies far outstrips our capacity to systematically study their functions. I will discuss our attempt to develop a scalable genetic screen approach, in vivo Perturb-Seq, and apply this method to the functional evaluation of 35 autism spectrum disorder (ASD) de novo loss-of-function risk genes. Using CRISPR-Cas9, we introduced frameshift mutations in these risk genes in pools, within the developing mouse brain in utero, followed by single-cell RNA sequencing of perturbed cells in the postnatal brain. We identified cell type-specific and evolutionarily conserved gene modules from both neuronal and glial cell classes. Recurrent gene modules and cell types are affected across this cohort of perturbations, representing key cellular effects across sets of ASD/ND risk genes. In vivo Perturb-Seq pilots a systems genetics approach to investigate at scale how diverse mutations affect cell types and states in the developing brain, and can be used in various in vivo and in vitro model organisms and systems.</p>
        <script type="text/JavaScript">
            update_visibility();
        </script>
    </body>
    </html>
    